napari/components/_viewer/model.py

```diff
@@ -230,7 +230,7 @@ def _update_layers(self):
         """Updates the contained layers.
         """
         for layer in self.layers:
-            layer._set_view_slice(self.dims.indices)
+            layer.indices = self.dims.indices
```

Comment:
what's our plan when we implement new slicing operations?

Reply:
i think we'll need to send more info over to the layers, but I like setting properties like this - or do you think this is an anti-pattern?

Reply:
not sure @jni do u wanna weigh in?

Reply:
I think the properties can be overwritten by the layers, and then they can do what they want, so it *should* be fine? ie in terms of expressivity they are equivalent. I'm certainly happy for us to leave this as is for now and revert *if* it becomes a problem.

Reply:
ok great - sounds good